middlewares/verifyToken.js

```diff
@@ -0,0 +1,18 @@
+function verifyToken(req, res, next) {
+  const bearerHeader = req.headers.authorization;
+  if (typeof bearerHeader !== 'undefined') {
```

Comment:
`if (bearerHeader)`

And if you want compare something with `undefined`, just `(bearerHeader !== undefined)`